### PR TITLE
Wokflow.md: Add unit usage and fix incorrect figures

### PR DESCRIPTION
**Pull request by DunklesArchipel**
Extended the documentation on how units should be provided in the labels (relevant for #95).

Replaced erroneous figures in the pdf and documentation (Addresses tasks in #88).

Includes documentation for #112.

### CODE REVIEW

File: doc/workflow.md [hunk 1]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
-For this example we use inkscape to digitize some data in plot 2a in the PDF, which is is located on page two of the manuscript (filename containing `_p1`). The plot contains three curves, which can be identified by their colors. Preferably each digitized curve should be placed in a single svg file. Therefore, create a copy of the SVG file of page two (`mustermann_2021_svgdigitizer_1_p1.svg`) and rename it to`mustermann_2021_svgdigitizer_1_p0_2b_blue.svg`, to indicate that this files contains the digitized curve of the blue curve in Figure 2a on page 2. 
+For this example we use inkscape to digitize some data in plot 2a in the PDF, which is is located on page two of the manuscript (filename containing `_p1`). The plot contains three curves, which can be identified by their colors. Preferably each digitized curve should be placed in a single svg file. Therefore, create a copy of the SVG file of page two (`mustermann_2021_svgdigitizer_1_p1.svg`) and rename it to`mustermann_2021_svgdigitizer_1_p1_f2a_blue.svg`. Here, (`_f2ba`) indicates that the digitized curve is in Fig. 2b  and that the curve is blue ( `_blue`) .  The latter identifier will later also be included in the SVG file.
```
Comment (saraedum) [inline doc/workflow.md:83]:
```suggestion
For this example we use inkscape to digitize some data in plot 2a in the PDF, which is located on page two of the manuscript (filename containing `_p1`). The plot contains three curves, which can be identified by their colors. Preferably each digitized curve should be placed in a single SVG file. Therefore, create a copy of the SVG file of page two (`mustermann_2021_svgdigitizer_1_p1.svg`) and rename it to`mustermann_2021_svgdigitizer_1_p1_f2a_blue.svg`. Here, `_f2a_blue` indicates that the digitized curve is in Fig. 2b  and that the curve is blue .  The identifier will later also be included in the SVG file.
```

File: doc/workflow.md [hunk 2]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
+
+ **2: Mark axis positions and labels**
 
- **2: Trace the curve**
+In the SVG file, by marking two positions on each axis, both the units and the scale of the axis will be extracted.
 
-1. Open the SVG file.
+In the example, the first position on the x-axis will be 0.0 V vs RHE. Add a text label which contains `x1: 0.0 V vs RHE`. Draw a line from connecting the text label and the position on the x-axis. Finally **group** the **line** and the **text label**.
```
Comment (saraedum) [inline doc/workflow.md:89]: from connecting?

File: doc/workflow.md [hunk 3]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
-   ![inkscape_x1_label](files/images/inkscape_x1_label.png)
+**Some notes on units and labels:**
 
-   **Some notes on labels:**
+* Units should be provided in the [astropy format](https://docs.astropy.org/en/stable/units/index.html). In brief:
+  * [Standard units](https://docs.astropy.org/en/stable/units/standard_units.html) simply read: `V`, `A` or `m`
+  * These can be preceeded by common prefixes, i.e., `mV`, `kV` or `uV`.
+  * Note that $\mu$ is `u`
+  * fractions read as follows (note the spaces between the dashes): `mV / s`, `mA / cm2`
+  * for square, cubic, etc units simply ass `2`, `3`, etc behind the unit (see line above)
```
Comment (saraedum) [inline doc/workflow.md:100]:
```suggestion
  * for square, cubic, etc units simply add `2`, `3`, etc. to the unit (see line above)
```

File: doc/workflow.md [hunk 4]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
-3. Mark two positions one for each axis (x and y). The first position on the x-axis will be 0.0 V vs RHE. Add a text label which contains `x1: 0.0 V vs RHE`. Draw a line from connecting the text label and the position on the x-axis. Finally group the line and the text label. Repeat this for positions `x2`, `y1` and `y2`.
+Repeat this for positions `x2`, `y1` and `y2`.
 
-   ![inkscape_x1_label](files/images/inkscape_x1_label.png)
+**Some notes on units and labels:**
 
-   **Some notes on labels:**
+* Units should be provided in the [astropy format](https://docs.astropy.org/en/stable/units/index.html). In brief:
+  * [Standard units](https://docs.astropy.org/en/stable/units/standard_units.html) simply read: `V`, `A` or `m`
+  * These can be preceeded by common prefixes, i.e., `mV`, `kV` or `uV`.
```
Comment (saraedum) [inline doc/workflow.md:97]:
```suggestion
  * Units can be preceeded by common prefixes, i.e., `mV`, `kV` or `uV`.
```

File: doc/workflow.md [hunk 5]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
+  * Note that $\mu$ is `u`
+  * fractions read as follows (note the spaces between the dashes): `mV / s`, `mA / cm2`
+  * for square, cubic, etc units simply ass `2`, `3`, etc behind the unit (see line above)
+  * Use brackets if more than one unit is in the numerator or denominator, i.e., `(kg m) / s2`
```
Comment (saraedum) [inline doc/workflow.md:101]:
```suggestion
  * Use parentheses if more than one unit is in the numerator or denominator, i.e., `(kg m) / s2`
```

Brackets are `[]`.

File: doc/workflow.md [hunk 6]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
+* If the reference on the voltage axis is not given in the plot. Extract this information from the manuscript text. Note that the reference scale given in the experimental section can be different to that in the main text of the manuscript.
```
Comment (saraedum) [inline doc/workflow.md:102]:
```suggestion
* If the reference on the voltage axis is not given in the plot, extract this information from the manuscript text. Note that the reference scale given in the experimental section can be different to that in the main text of the manuscript.
```

File: doc/workflow.md [hunk 7]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
+* `svgdigitizer` only considers units in the labels in positions`x2` and `y2`.
```
Comment (saraedum) [inline doc/workflow.md:103]:
```suggestion
* `svgdigitizer` only considers units in the labels for positions`x2` and `y2`. 
```

File: doc/workflow.md [hunk 8]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
+* `svgdigitizer` only considers units in the labels in positions`x2` and `y2`. 
 
-   **TODO #86:** Add link to possible typos.
+![inkscape_x1_label](files/images/inkscape_x1_label.png)
 
-   * In principle the units have to be provided only on the `x2` and `y2` position. 
-   * $\mu$ can be used, but it is more convenient to use `u`instead.
-   * $^{-2}$ can be written as `-2`
-   * The reference on the voltage axis is not always given in the plot. Extract this information from the manuscript text.
+When all axes have been marked the plot looks like the following:
```
Comment (saraedum) [inline doc/workflow.md:107]:
```suggestion
When all axes have been marked, the plot looks like the following:
```

File: doc/workflow.md [hunk 9]
```diff
@@ -113,21 +122,25 @@ For this example we use inkscape to digitize some data in plot 2a in the PDF, wh
...
-5. Add a text field next and name it `curve: identifier`, which in our case would be `curve: blue`.
+2. Add a text field next and name it `curve: identifier`, which in our case would be `curve: blue`.
 
-6. Group the text field and the curve.
+3. Group the text field and the curve.
 
 ![inkscape_smoothed_path_curve](files/images/inkscape_smoothed_path_curve.png)
 
-7. Add a text field to the plot containing the scan rate, with which the data was acquired. This is not necessarily given in the plot or figure description and might have to extracted from the text of the manuscript: `scan rate: 50 mV / s`.
+7. Add a text field to the plot containing the scan rate, with which the data was acquired. This value is not necessarily given in the plot or figure description and might have to be extracted from the text of the manuscript. Here `scan rate: 50 mV / s`.
```
Comment (saraedum) [inline doc/workflow.md:131]:
```suggestion
7. Add a text field to the plot containing the scan rate with which the data was acquired. This value is not necessarily given in the plot or figure description and might have to be extracted from the text of the manuscript. Here, `scan rate: 50 mV / s`.
```

File: doc/workflow.md [hunk 10]
```diff
@@ -113,21 +122,25 @@ For this example we use inkscape to digitize some data in plot 2a in the PDF, wh
...
+8. Feel free to add a comment (full sentence) to the plot, which will later be included in the metadata file and will be printed on the website. For example, `comment: The curve is a bit noisy.`
```
Comment (saraedum) [inline doc/workflow.md:132]:
```suggestion
8. Feel free to add a comment to the plot. This comment will later be included in the metadata file and will be shown on the website. Use a full sentence, for example, `comment: The curve is a bit noisy.` 
```

File: doc/workflow.md [hunk 11]
```diff
@@ -113,21 +122,25 @@ For this example we use inkscape to digitize some data in plot 2a in the PDF, wh
...
 The final file should look like this:
 
 ![inkscape_final](files/images/inkscape_final.png)
 
 ## Step 4: Create a metadata file for each digitized curve
 
-Create a YAML file with the same name than the SVG file: `mustermann_2021_svgdigitizer_1_p0_2b_blue.yaml`
+Create a YAML file with the same name than the SVG file: `mustermann_2021_svgdigitizer_1_p0_f2b_blue.yaml`
+
+
```
Comment (saraedum) [inline doc/workflow.md:143]:
```suggestion
Create a YAML file with the same name as the SVG file: `mustermann_2021_svgdigitizer_1_p0_f2b_blue.yaml`
```

**Review by saraedum — APPROVED** [review]

**Comment by saraedum** [comment]
GitHub won't show the updated .png files. (It recommends to try again later.) So I did not check these.

File: doc/workflow.md (hunk 5 above)
Comment (DunklesArchipel) [reply doc/workflow.md:101]: sure 😵‍💫 :)

File: doc/workflow.md [hunk 12]
```diff
@@ -78,30 +80,37 @@ The filenames are of the form:
...
+For this example, we use inkscape to digitize some data in plot 2a in the PDF, which is located on page two of the manuscript (filename containing `_p1`). The plot contains three curves, which can be identified by their colors. Preferably each digitized curve should be placed in a single SVG file. Therefore, create a copy of the SVG file of page two (`mustermann_2021_svgdigitizer_1_p1.svg`) and rename it to`mustermann_2021_svgdigitizer_1_p1_f2a_blue.svg`. Here, `_f2a_blue` indicates that the digitized curve is in Fig. 2b and that the curve is blue. The identifier will later also be included in the SVG file.
+
+ **2: Mark axis positions and labels**
 
- **2: Trace the curve**
+In the SVG file, by marking two positions on each axis, both the units and the scale of the axis will be extracted.
 
-1. Open the SVG file.
+In the example, the first position on the x-axis will be 0.0 V vs RHE. Add a text label which contains `x1: 0.0 V vs RHE`. Draw a line from connecting the text label and the position on the x-axis. Finally **group** the **line** and the **text label**.
```
Comment (DunklesArchipel) [inline doc/workflow.md:89]:
```suggestion
In the example, the first position on the x-axis will be 0.0 V vs RHE. Add a text label that contains `x1: 0.0 V vs RHE`. Draw a line connecting the text label and the position on the x-axis. Finally **group** the **line** and the **text label**. 
```

**Comment by DunklesArchipel** [comment]
> GitHub won't show the updated .png files. (It recommends to try again later.) So I did not check these.

I can see the updated files in `files changed`. They seem fine.

**Pull request merged**